java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1766,15 +1767,27 @@ public T persist(@NonNull String key) {
      *
      * @see <a href="https://redis.io/commands/time/">redis.io</a> for details.
      * @return Command Response - The current server time as a <code>String</code> array with two
-     *     elements: A Unix timestamp and the amount of microseconds already elapsed in the current
-     *     second. The returned array is in a <code>[Unix timestamp, Microseconds already elapsed]
+     *     elements: A <code>UNIX TIME</code> and the amount of microseconds already elapsed in the
```

Comment:
"A UNIX TIME" sounds a little more awkward than "A Unix timestamp", but I think it's still acceptable. We'll see what they think upstream.